### ISSUE TITLE
..is it possible to open a Vault per Default?

### ISSUE THREAD

**Issue by Naboo2604** [elize1979/AzureKeyVaultExplorer#14]
Hi,
very cool tool - thanks for that!

one question: is it possible to make a vault as "default" ? - this vault should be openened after starting the tool without choosing one of the vaults in drop-down-menu...

would be great having this option.

thanks and greets, 
Markus

**Comment by guestdj** [comment]
Yes, I would also love to see this option. It's a great app, just would save some time if we could select a vault from the Vaults drop down

**Comment by guestdj** [comment]
Figured this out, you need to update the VaultAliases.json to specify your vault name:
[
  {
    "Alias": "Name-of-vault",
    "VaultNames": [ "key-vault-name-in-azure" ]
  }
]
You also need to update Vaults.json:
{
  "Name-of-vault": { 
    "$id": "1",
    "ReadWrite": [
      {
        "$type": "Microsoft.Vault.Library.VaultAccessUserInteractive, Microsoft.Vault.Library",
        "DomainHint": "microsoft.com",
        "UserAliasType": "Alt"
      }
    ],
    "ReadOnly": [
      {
        "$type": "Microsoft.Vault.Library.VaultAccessUserInteractive, Microsoft.Vault.Library",
        "DomainHint": "microsoft.com",
        "UserAliasType": "Alt"
      }
    ]
  }
}
